src/providers/auto-complete/contacts-service.ts

```diff
@@ -6,42 +6,60 @@ import lodash from 'lodash';
 import { UserDataProvider } from '@providers/user-data/user-data';
 import { PublicKey } from 'ark-ts/core';
 
+export class AutoCompleteContact {
```

Comment:
Use an interface instead and move it to `models/contact` to keep the standard.

Reply:
Why an interface? Because what I don't like about is is, that have to write the object myself:

```    const contacts = lodash.map(this.userDataProvider.currentProfile.contacts, (value, key) => {
      if (value['name']) {
        return new AutoCompleteContact(key.toString(), value['name'].toString(), "ios-contacts-outline");
      }
    });
```
vs
```
   const contacts: AutoCompleteContact[] = lodash.map(this.userDataProvider.currentProfile.contacts, (value, key) => {
      if (value['name']) {
        return {address: key.toString(), name: value['name'].toString(), iconName: "ios-contacts-outline"};
      }
    });
``` 

Or is is just that we should always use interfaces when the class has no logic?